lib/java/src/org/apache/thrift/server/TThreadPoolServer.java

```diff
@@ -173,13 +171,14 @@ public void serve() {
   	}
 
   	execute();
-  	waitForShutdown();
-    
+  	if (!waitForShutdown()) {
+  	  LOGGER.error("Shutdown is not done after " + stopTimeoutVal + stopTimeoutUnit);
+    }
```

Comment:
whitespace looks off here?

Reply:
In fact, it seems the closing bracket is correctly indented (with white spaces).
The problem is from previsous lines (especially the existent code base), it is using tabulation instead of white space for indentation.
I'll fix my code here and then eventually push another PR for fixing existent code base.